service/model/project.go

```diff
@@ -193,3 +197,7 @@ func toGithubRepository(rawURL string) (GithubRepository, error) {
 		RepositorySlug: slugs[1],
 	}, nil
 }
+
+type GitTag struct {
+	Name string
+}
```

Comment:
GitHub API returns only the git tag name, but this is sufficient. From the user's perspective, they will simply select a tag to link the release to the source code. When creating a GitHub release, GitHub requires only the git tag name, so no additional parameters are needed